src/client/App.jsx

```diff
@@ -14,16 +20,54 @@ function App() {
       .then(res => setMovies(res.data));
   }, []);
 
-  const handleRegister = async ({ username, password }) => {
-    
+  const handleRegister = async user => {
+    fetch(`${apiUrl}/user/register`, {
+      ...headers,
+      body: JSON.stringify(user)
+    })
+    .then(res => res.json())
+    .then(json => {
+      console.log(json)
+      if (json.error) {
+        window.alert(`${json.error}!`)
```

Comment:
Like the inclusion of alerts. Although it's a small implementation it shows you're thinking about display in a real life application as opposed to just meeting the requirements